tests/integrations/opentelemetry/test_span_processor.py

```diff
@@ -242,6 +250,7 @@ def test_update_span_with_otel_status(otel_status, expected_status):
     assert sentry_span.get_trace_context().get("status") == expected_status
 
 
+@pytest.mark.forked
```

Comment:
Just wondering, why do we need to be forking so many tests?

Reply:
Our test infrastructure has a lof of code for configuring how tests are run (see `conftest.py` files). Some of this is tailored to the hub based system and does cleaning up of hubs between tests, but not really for the new scopes. 

With this forked approach we just make sure that each test gets its own process so data is not leaked in between tests. At some point we should take the time to look all the patching going on in conftest.py and clean it up.